RevenueCatUI/Resources/en.lproj/Paywall.strings

```diff
@@ -0,0 +1,13 @@
+"iPad.step1.title" = "Today: Free trial for 7 days";
+"iPad.step1.subtitle" = "Get full access to design and animation tools";
+"iPad.step2.title" = "Day 5: Reminder";
+"iPad.step2.subtitle" = "We’ll send you a reminder 2 days before your free trial ends.";
+"iPad.step3.title" = "Day 7: Trial ends";
+"iPad.step3.subtitle" = "Your Pro subscription starts, unless you’ve canceled during the trial.";
+
+"iPhone.step1.title" = "Start free trial";
+"iPhone.step1.subtitle" = "Today";
+"iPhone.step2.title" = "Reminder";
+"iPhone.step2.subtitle" = "Day 5";
+"iPhone.step3.title" = "Trial ends";
+"iPhone.step3.subtitle" = "Day 7";
```

Comment:
but how are we going to translate it? our translation setup is only in vectornator github project.
I was planning to read it from Bundle.main.localization(key:., table: `paywall`)

Reply:
I've changed to use the same thing, lets revisit it later